openmetadata-ui/src/main/resources/ui/src/components/Modals/IngestionRunDetailsModal/IngestionRunDetailsModal.tsx

```diff
@@ -88,24 +88,28 @@ function IngestionRunDetailsModal({
     () => ({
       expandedRowRender: (record) => {
         return (
-          record.failures?.map((failure) => (
-            <ConnectionStepCard
-              isTestingConnection={false}
-              key={failure.name}
-              testConnectionStep={{
-                name: failure.name,
-                mandatory: false,
-                description: failure.error,
-              }}
-              testConnectionStepResult={{
-                name: failure.name,
-                passed: false,
-                mandatory: false,
-                message: failure.error,
-                errorLog: failure.stackTrace,
-              }}
-            />
-          )) ?? []
+          <Row gutter={[16, 16]}>
+            {record.failures?.map((failure) => (
+              <Col key={failure.name} span={24}>
+                <ConnectionStepCard
+                  isTestingConnection={false}
+                  key={failure.name}
+                  testConnectionStep={{
+                    name: failure.name,
+                    mandatory: false,
+                    description: failure.error,
+                  }}
+                  testConnectionStepResult={{
+                    name: failure.name,
+                    passed: false,
+                    mandatory: false,
+                    message: failure.error,
+                    errorLog: failure.stackTrace,
+                  }}
+                />
+              </Col>
+            )) ?? []}
```

Comment:
@ShaileshParmar11,as we are returning the JSX I believe we don’t need this empty array as a fallback.

Moreover, I believe we should return null if there are no items to render.